src/components/file/FileInfoModal.tsx

```diff
@@ -120,70 +135,127 @@ const FileInfoModal: FC<Props> = (props) => {
         }
 
         if (isImage) {
-            return <Image fileId={fileInfo.id} maxSize={300} alt={fileInfo.name} />;
+            return (
+                <Box mx="auto" my={2} width="fit-content">
+                    <Image fileId={fileInfo.id} maxSize={300} alt={fileInfo.name} />;
```

Comment:
Please take care not to leave semicolon in the JSX because they get rendered and look strange. You also left one in the Spinner but I removed that before. Please try to avoid it in the future :)
```suggestion
                    <Image fileId={fileInfo.id} maxSize={300} alt={fileInfo.name} />
```